chsdi/models/vector/uvek.py

```diff
@@ -488,64 +488,42 @@ class ENERGIESTAEDTE2000AUFDEMWEG(Base, Vector):
 
 
 class ENERGIEFORSCHUNG(Base, Vector):
-    __tablename__ = 'energieforschung'
+    __tablename__ = 'energieforschung_new'
```

Comment:
Is this a new layer? Or replacing existing layer?

Does that mean that old links with the old technical name won't work anymore?